apps/src/studio/rocketHeightLogic.js

```diff
@@ -0,0 +1,51 @@
+var CustomGameLogic = require('./customGameLogic');
+var studioConstants = require('./constants');
+var Direction = studioConstants.Direction;
+var codegen = require('../codegen');
+var api = require('./api');
+
+/**
+ * Custom logic for the Rocket Height levels
+ * @constructor
+ * @implements CustomGameLogic
+ */
+var RocketHeightLogic = function (studio) {
+  CustomGameLogic.apply(this, arguments);
+  this.rocketIndex = 0;
+  this.rocket = null;
+  this.last = Date.now();
+  this.seconds = 0;
+  this.height = 0;
+};
+RocketHeightLogic.inherits(CustomGameLogic);
+
+RocketHeightLogic.prototype.onTick = function () {
+  
+  // Update the rocket once a second
+  if (Date.now() - this.last < 1000) {
+    return;
+  } else {
```

Comment:
second part doesnt need to be inside an else, since we've either returned or we havent.